.tools/bin/update-all

```diff
@@ -10,6 +10,10 @@ rm -rf redaxo/src/core/vendor/filp/whoops/.github/
 rm -rf redaxo/src/core/vendor/psr/log/Psr/Log/Test/
 rm -rf redaxo/src/core/vendor/ramsey/http-range/tests/
 rm -rf redaxo/src/core/vendor/symfony/console/Tests/
+rm -rf redaxo/src/core/vendor/symfony/http-foundation/RateLimiter/
+rm -rf redaxo/src/core/vendor/symfony/http-foundation/Session/
+rm -rf redaxo/src/core/vendor/symfony/http-foundation/Test/
+rm -f redaxo/src/core/vendor/symfony/http-foundation/*Response*.php
```

Comment:
Ich habe den Session-Code und die Response-Klassen entfernt.

Ich finde es immer bisschen kritisch, so direkt Klassen im Vendor zu löschen.
Einerseits, da man sich nicht 100% sicher ist, ob es ohne die klappt. Denke in diesem Fall aber eigentlich schon.
Und andererseits hat man halt eine halbe Lib. Wenn jemand im Frontend zum Beispiel einen anderen Part aus der Lib verwenden möchte, dann gibt es ein Problem, da der Part nicht da ist, aber die Lib komplett erneut laden eventuell auch problematisch wäre. Allerdings glaube ich eigentlich nicht, dass jemand selbst hier etwas mit den fehlenden Klassen machen wollen wird.
Und ansonsten kann man sie ja, wenn der Wunsch aufkommt, dann hinzunehmen.

Ich wäre andersrum aber auch einverstanden, wenn wir doch die ganze Lib reinnehmen würden (also nur Tests excluden). 
Kann mich also nicht recht entscheiden, und bin mit beiden Varianten einverstanden.

Reply:
Wieviel dateien sparen wir dadurch?

Ne halbe lib fänd ich auch schwierig

Reply:
6 Dateien bzgl. Response
2 Dateien bzgl. RateLimiter
31 Dateien bzgl. Session, durch die verschiedenen Storages/Handler: https://github.com/symfony/http-foundation/tree/5.x/Session

Ich glaube, ich würde es dann doch eher komplett aufnehmen.

Reply:
> Ich glaube, ich würde es dann doch eher komplett aufnehmen.

jop, mach das doch